src/Stormpath/Authc/UsernamePasswordRequest.php

```diff
@@ -22,12 +22,15 @@ class UsernamePasswordRequest implements AuthenticationRequest
     private $username;
     private $password;
     private $host;
+    private $accountStore;
 
-    public function __construct($username, $password, $host = null)
+    public function __construct($username, $password, array $options = array())
```

Comment:
This will probably breaks backwards compatibility, but I don't think people are using the "host" argument anyways.